src/scripts.js

```diff
@@ -320,6 +377,16 @@ const displayResults = (searchInput, recipes) => {
   }
 }
 
+const displayFavorites = () => {
+  displayRecipeList();
```

Comment:
We'll need to tweak this function a bit to merge it with what I'm doing. I built a separate Favorites view so that we can separate that favorites search button from the general search button.